openmetrics/tests/conftest.py

```diff
@@ -0,0 +1,55 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+
+import mock
+import pytest
+from prometheus_client import CollectorRegistry, Counter, Gauge, generate_latest
+
+from datadog_checks.base import ensure_unicode
+from datadog_checks.dev import docker_run, get_docker_hostname
+
+HERE = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Let's use `get_here` from `datadog_checks.dev`

Reply:
Good advice, this should be everywhere!